pyproject.toml

```diff
@@ -80,8 +80,6 @@ exclude = [
 "vllm/model_executor/models/**/*.py" = ["UP006", "UP035"]
 "vllm/prompt_adapter/**/*.py" = ["UP006", "UP035"]
 "vllm/spec_decode/**/*.py" = ["UP006", "UP035"]
-"vllm/worker/**/*.py" = ["UP006", "UP035"]
-"vllm/utils.py" = ["UP006", "UP035"]
```

Comment:
This was added in https://github.com/vllm-project/vllm/pull/14296 to fix an error with Cython which ROCm depends on

Reply:
oh.. thank you for your remind.. so all done?

Reply:
We should allow `utils.py` to use `Type` for now.